cvat/apps/engine/models.py

```diff
@@ -30,7 +30,7 @@ class Task(models.Model):
     bug_tracker = models.CharField(max_length=2000, default="")
     created_date = models.DateTimeField(auto_now_add=True)
     updated_date = models.DateTimeField(auto_now_add=True)
-    status = models.CharField(max_length=32, default="annotate")
+    status = SafeCharField(max_length=32, default="annotation")
```

Comment:
I believe you don't want the change. It is our internal error if we try to put here something more than 32. Better way to implement that is using Enums: https://hackernoon.com/using-enum-as-model-field-choice-in-django-92d8b97aaa63